### DISCUSSION TITLE
Install stuck on "Checking signature..."

### DISCUSSION THREAD

**Discussion by MrMewlax** [rapid7/metasploit-framework#14581] [Q&A] [answered]
After installing I can only get up to here, can anyone help?
<img width="567" alt="Screenshot 2021-01-04 at 15 28 48" src="https://user-images.githubusercontent.com/64780964/103550958-8b60a500-4ea1-11eb-9b57-8c0513341edb.png">

**Comment by adfoster-r7** [comment, marked as answer]
Hi there, this looks closer to be local issue rather than a problem specific to Metasploit

Please raise an issue if you're still running into difficulties:
https://github.com/rapid7/metasploit-framework/issues/new/choose
